### DISCUSSION TITLE
Differences between SW 20200101.8 and 20200101.7

### DISCUSSION THREAD

**Discussion by cyress1** [solokeys/solo2#94] [Q&A]
Hi, 
i'm trying to use my new solokeys.
only two of them are working with 2FA for Google and tutanota.com (SW .7)

The two with V .8 are not working with Google and tutanote, but with GitHub they are working.
For example on tutanota.com i get the error message "Your stick was not recognized"

Can anybody confirm this issue?

**Comment by joshuarubin** [comment]
I'm unable to use the .8 firmware with any site in any browser on linux. Tried firefox, chrome and chromium.

**Comment by nickray** [comment]
I'm afraid I can't answer this question, as I was not involved in cutting those releases. However, now that the CLI was extended in v0.1.1 to support updates on macOs + Debian (so a majority of owners can actually do the update), plan to cut a new release with the two important fixes so far, possibly waiting for https://github.com/solokeys/fido-authenticator/pull/8.

**Comment by joshuarubin** [comment]
Upgraded to .9 and still can't use the key with Firefox on Linux (arch). It did work on .7.

**Reply by PeterFalken** [reply]
I was able to update to .9 as well - but it's unclear what has changed between .7, .8 and .9

**Reply by cyress1** [reply]
I'm using Vivaldi on Linux Mint.
Solo v.2 sticks with v.9 are working with Google, Nextcloud and GitHub
One thing that is a little bit annoyng is the touch of the solo, sometimes i really have to Grab the stick between two fingers, to get it triggerd.

But whats me also confusing, my solo with USB-A change the color when triggered, the other ones with USB-C are only flashing brighter

**Reply by data-henrik** [reply]
I tested my USB-C on RHEL 8.5 and have trouble, too. The NFC works on Android with a FIDO2 /Webauthn test app
